### PR TITLE
Update command

### PR DESCRIPTION
**Pull request by kirtangajjar**

### CODE REVIEW

File: php/commands/src/CLI_Command.php [hunk 1]
```diff
@@ -179,35 +179,158 @@ public function info( $_, $assoc_args ) {
 	 * ## EXAMPLES
 	 *
 	 *     # Update CLI.
-	 *     $ ee cli update
 	 *     You have version 0.24.0. Would you like to update to 0.24.1? [y/n] y
-	 *     Downloading from https://github.com/ee/ee/releases/download/v0.24.1/ee-0.24.1.phar...
+	 *     Downloading from https://github.com/ee/ee/releases
```
Comment (mrrobot47) [inline php/commands/src/CLI_Command.php]: Update the url to proper easyengine release url

File: php/commands/src/CLI_Command.php [hunk 2]
```diff
@@ -179,35 +179,158 @@ public function info( $_, $assoc_args ) {
...
+	 *     $ ee cli update/download/v0.24.1/ee-0.24.1.phar...
```
Comment (mrrobot47) [inline php/commands/src/CLI_Command.php]: Make the versions start from 4.

File: php/commands/src/CLI_Command.php [hunk 3]
```diff
@@ -179,35 +179,158 @@ public function info( $_, $assoc_args ) {
...
+			$response = Utils\http_request( 'GET', $version_url );
+			if ( ! $response->success || 200 !== $response->status_code ) {
+				EE::error( sprintf( 'Failed to get current nightly version (HTTP code %d)', $response->status_code ) );
+			}
+			$nightly_version = trim( $response->body );
+			if ( EE_VERSION != $nightly_version ) {
+				$updates['nightly'] = array(
+					'version'        => $nightly_version,
+					'update_type'    => 'nightly',
+					'package_url'    => 'https://raw.githubusercontent.com/EasyEngine/easyengine-builds/master/phar/easyengine-nightly.phar	',
```
Comment (mrrobot47) [inline php/commands/src/CLI_Command.php]: Space in url? Please check this again.

File: php/EE/Runner.php [hunk 4]
```diff
@@ -629,6 +636,12 @@ private function set_alias( $alias ) {
 
 	public function start() {
 
+		// // Protect 'cli info' from most of the runtime,
+		// // except when the command will be run over SSH
```
Comment (mrrobot47) [inline php/EE/Runner.php]: Following phpcs wherever possible would be preferred.

File: php/commands/src/CLI_Command.php [hunk 5]
```diff
@@ -179,35 +179,158 @@ public function info( $_, $assoc_args ) {
...
+			$updated_version = 'the latest nightly release';
+		} elseif ( Utils\get_flag_value( $assoc_args, 'stable' ) ) {
+			$updated_version = 'the latest stable release';
+		} else {
+			$updated_version = $newest['version'];
+		}
+		EE::success( sprintf( 'Updated WP-CLI to %s.', $updated_version ) );
+	}
+
+		/**
```
Comment (mrrobot47) [inline php/commands/src/CLI_Command.php]: Following phpcs wherever possible would be preferred.

File: php/EE/Runner.php (hunk 4 above)
Comment (kirtangajjar) [reply php/EE/Runner.php]: Done.

File: php/commands/src/CLI_Command.php (hunk 5 above)
Comment (kirtangajjar) [reply php/commands/src/CLI_Command.php]: Done.

File: php/commands/src/CLI_Command.php (hunk 3 above)
Comment (kirtangajjar) [reply php/commands/src/CLI_Command.php]: Done.

File: php/commands/src/CLI_Command.php (hunk 2 above)
Comment (kirtangajjar) [reply php/commands/src/CLI_Command.php]: Done.

File: php/commands/src/CLI_Command.php (hunk 1 above)
Comment (kirtangajjar) [reply php/commands/src/CLI_Command.php]: Done.

**Pull request merged**